src/js/helpers/mapHelper.js

```diff
@@ -475,3 +474,46 @@ export const performZIPGeocode = (zip) => {
         }
     };
 };
+
+export const getCitySearchRequestObj = (
```

Comment:
@MaxwellKendall I would move this into the dataMapping folder. I would prefer the helpers to just be fetching.
You could do something like 
`
export const getCitySearchRequestObj = (searchText= "", limit= 40, filter) => {
  return {
    search_text: searchText,
    limit,
    filter: Object.assign({}, ...filter)
  }
};
`

or, 

`
import { pickBy, identity } = 'lodash';

export const getCitySearchRequestObj = (searchText= "", state='', country='', scope='') => {
  return pickBy({
    search_text: searchText,
    limit: 40,
    filter: {
                country_code: country,
                scope,
                state_code: state
            }
  }, identity)
};
`